addons/html_builder/static/src/core/plugins/overlay_buttons/overlay_buttons_plugin.js

```diff
@@ -7,7 +7,12 @@ import { OverlayButtons } from "./overlay_buttons";
 export class OverlayButtonsPlugin extends Plugin {
     static id = "overlayButtons";
     static dependencies = ["selection", "overlay", "history", "operation"];
-    static shared = ["hideOverlayButtons", "showOverlayButtons"];
+    static shared = [
+        "hideOverlayButtons",
+        "showOverlayButtons",
+        "hideOverlayButtonsUi",
+        "showOverlayButtonsUi",
```

Comment:
I heard you talking with FGE, why don't you just call showOverlayButtons and hideOverlayButtons ? I don't see why we need to add two new functions that do the same :wink:
The presence of the buttons is given by the state.buttons if it is empty or not, and the fact that they are shown or not is given by state.isVisible, we don't need more I think :slightly_smiling_face: Unless you has an issue ? In that case, can you tell me what it was ? :slightly_smiling_face: